src/navigation/breadcrumbs/breadcrumbs.jsx

```diff
@@ -200,15 +233,26 @@ function Breadcrumbs(props) {
         `${BEM_NAVIGATION_BREADCRUMBS}--breadcrumb_title_typography`,
     );
 
+    const shouldUseStaticBreadcrumbs = Array.isArray(staticCrumbs);
```

Comment:
So `staticCrumbs` is prop-type-checked, and it defaults to an empty array.  What condition are we really checking for here?  Do we need it to be a _non-empty_ array, for instance?

Reply:
I forgot to remove this condition.

Reply:
Looks to be removed.  Resolving.